src/dbt_score/formatters/human_readable_formatter.py

```diff
@@ -33,18 +33,30 @@ def model_evaluated(
         """Callback when a model has been evaluated."""
         if score.value < self._config.fail_any_model_under:
             self._failed_models.append((model, score))
-        print(f"{score.badge} {self.bold(model.name)} (score: {score.rounded_value!s})")
-        for rule, result in results.items():
-            if result is None:
-                print(f"{self.indent}{self.label_ok} {rule.source()}")
-            elif isinstance(result, RuleViolation):
-                print(
-                    f"{self.indent}{self.label_warning} "
-                    f"({rule.severity.name.lower()}) {rule.source()}: {result.message}"
-                )
-            else:
-                print(f"{self.indent}{self.label_error} {rule.source()}: {result!s}")
-        print()
+        if (
+            score.value < self._config.fail_any_model_under
+            or any(isinstance(result, RuleViolation) for result in results.values())
+            or self._config.show_all
```

Comment:
I think this is not 100% correct :thinking: If I understand correctly we want to print the following output if:

`score.value < self._config.fail_any_model_under`. Then, only show the failing rules. Now it will also show the failing rules of models that did not fail

Reply:
Ok I got it! :+1:  I think for the scenario you describe it is indeed useful to be able to show all the failing rules and we can definitely leave that as the default. I do think that we should also have the option to show only failing models, with their failing rules.

Maybe we should have two flags: `--show-all-rules` and `--show-all-models` so the user is able to further specify the output. @matthieucan curious to hear your opinion as well!

So then the user is able to:
1. show failing models, with all rules
2. show failing models, with failing rules
3. show all models, with all rules
4. show all models, with failing rules

Reply:
Good to align on those expectations indeed.
Maybe the `--show` parameter could take an argument, e.g.
- `--show all` - show all models, all rules
- `--show failing-models` - show failing rules of failing models
- `--show failing-rules` - show failing rules of all models (the default?)

I'm not sure if the first scenario mentioned (show failing models, with all rules) is useful, considering the option to use `--select` in combination. For example `--select my_model --show all` might be more actionable. But let me know what you think :)

Reply:
You are right that option 1 (failing models with all rules) is probably not very useful. I think the direction of `--show something` would be a nice one. It's indeed simpler than providing two flags. And agreed that `--show failing-rules` should be the default!

Reply:
Sounds good and happy to adjust it in the next few days - thanks for all the input.

Reply:
In terms of wording: The codebase often refers to "violated" rules, e.g. `RuleViolation` and not failing rules. What do you think of:
`--show all`
`--show failing-models`
`--show violated-rules`
?

Reply:
I think it's simpler to remember for users if the same term is used for the CLI options, as it's an abstraction over the code base

Reply:
Hi @jochemvandooren  sorry not picking it up earlier! I initially planned to work on it much sooner again. I am picking it up today evening again and will come back to you asap then - hope that works for you.

Reply:
I just added the different options as discussed above. Will now work on the merge conflicts :)

Reply:
No problem @thomend ! Was afraid you had forgotten :cry: I will review tomorrow!